### ISSUE TITLE
IJ build 771 failure

### ISSUE THREAD

**Issue by SlimeDog** [RockinChaos/ItemJoin#437]
## Thanks for taking the time to open a bug report!

**Describe the bug**
IJ b771 does not open a DeluxeMenus menu. It works fine with IJ b770.

**To Reproduce**
1. Start server with IJ b771
2. Click on IJ item that should open DeluxeMenus menu
3. Observe that menu does not open

**Expected behavior**
Open menu.

**Screenshots**
N/A

**Pastebin**
config.yml sans # comments and blank lines
```
config-Version: 8
Language: 'English'
General:
  CheckforUpdates: false
  Metrics-Logging: false
  Log-Coloration: false
  Log-Commands: true
  Debugging: false
Database:
  MySQL: false
  database: 'database_name'
  prefix: 'ij_'
  host: 'localhost'
  port: 3306
  user: 'root'
  pass: 'password'
Settings:
  HeldItem-Slot: 0
  HeldItem-Triggers: JOIN, RESPAWN, WORLD-SWITCH
  HeldItem-Animations: false
  Default-Triggers: JOIN, RESPAWN, WORLD-SWITCH
  DataTags: true
Permissions:
  Obtain-Items: false
  Obtain-Items-OP: false
  Commands-Get: false
  Commands-OP: false
  Movement-Bypass: false
Clear-Items:
  Type: ITEMJOIN
  Delay-Tick: 2
  Join: false
  Quit: false
  World-Switch: false
  Region-Enter: false
  Options: PROTECT, PROTECT_OP, PROTECT_CREATIVE
  Blacklist: ''
Active-Commands:
  commands:
    - 'itemjoin get <item> <player>'
    - 'first-join: say This is a command only executed once per world, per player.'
  commands-sequence: SEQUENTIAL
  triggers: JOIN
  enabled-worlds: DISABLED
Prevent:
  Pickups: false
  itemMovement: false
  Self-Drops: false
  Death-Drops: false
  Bypass: DISABLED
softDepend:
  Multiverse-Core: true
  Multiverse-Inventories: true
  PlaceholderAPI: true
  Vault: true
  WorldGuard: true
  AuthMe: false
  BetterNick: false
  HeadDatabase: false
  MyWorlds: false
  PerWorldInventory: false
  PerWorldPlugins: false
  TokenEnchant: false
  xInventories: false
```
items.yml sans # comments
```
items-Version: 8

items-Delay: 2
items-Overwrite: true
items-Spamming: false
items-RestrictCount: true

items:
  changeWorld-tool:
    id: CLOCK
    slot: 8
    count: 1
    name: '&bNavigate'
    lore:
      - 'Teleport elsewhere'
    interact-right:
    - 'player: deluxemenus open navigation4'
    itemflags: cancel-events, death-drops, hide-attributes, hide-durability, inventory-modify, placement, self-drops, unbreakable
    permission-node: 'deluxemenus.open'
    # disabled-worlds-UNSUPPORTED: world_void
    enabled-worlds: world, world_lobby, world_blackdog, world_city, world_luckyblock, world_minigames, world_skyblock, world_westeros, world_whitedog, world_yellowdog
```

**Please complete the following information:**
```
>version
[12:01:47] [Server thread/INFO]: This server is running CraftBukkit version 3278-Spigot-3892929-2071671 (MC: 1.17.1) (Implementing API version 1.17.1-R0.1-SNAPSHOT)
[12:01:47] [Server thread/INFO]: You are running the latest version
> version ItemJoin
[12:01:58 INFO]: ItemJoin version 5.2.3-SNAPSHOT-b771
[12:01:58 INFO]: Get custom items on join.
[12:01:58 INFO]: Website: https://www.spigotmc.org/resources/itemjoin.12661/
[12:01:58 INFO]: Author: RockinChaos
>version DeluxeMenus
[12:02:19] [Server thread/INFO]: DeluxeMenus version 1.13.4-DEV-119
[12:02:19] [Server thread/INFO]: All in one inventory menu system
[12:02:19] [Server thread/INFO]: Authors: extended_clip and Glare
```

**Comment by SlimeDog** [comment]
I suppose this could be an issue with configuration, but I saw no configuration changes between b770 and b771.

**Comment by FakEEE14** [comment]
same here build 770 work fine and give item on join
but build 771 need /ij reload to work properly after server reboot

**Comment by SlimeDog** [comment]
I confirm that b771 works after `ij reload`. So there is some issue with initialization.

**Comment by RockinChaos** [comment]
Confirmed and is now fixed in b772. I removed b771 since it was basically non-functional.
The reason for the error was because I changed the timings of ItemJoin to run faster and be more efficient when loading. Turns out I accidentally removed a (purposeful) delay that handled the registering of worlds into ItemJoin. Since the plugin runs at startup (preworld) I can only fetch the worlds (postworld) once startup is almost completed. Also these performance improvements were done mainly because of the new toggle command options as commands can only have tab completion if they are registered postworld with no delays.

Link; https://ci.craftationgaming.com/job/ItemJoin/772/
**Remember to remove your old ItemJoin.jar file**

Let me know~!
